### ISSUE TITLE
Router detected as vulnerable while its not

### ISSUE THREAD

**Issue by sasatefa2009** [threat9/routersploit#305]
Greetings,

My router model is " ZXHN H108N V2.5 ", it was detected as vulnerable to "exploits/routers/zte/zxv10_rce" while it was not.
![screenshot from 2017-09-06 16-48-49](https://user-images.githubusercontent.com/22708418/30118712-02f1361c-9324-11e7-904a-dfdb1b76fcb6.png)

**Comment by lucyoa** [comment]
It seems that exploit was able to authenticate successfully using hardcoded credentials but then failed with exploiting rce vulnerability.

Can you confirm that you were able to authenticate using: "root" / "W!n0&oO7." credentials?

**Comment by sasatefa2009** [comment]
@lucyoa  how could i confirm that ?

**Comment by lucyoa** [comment]
Visit web interface on port 80 and try to authenticate using: "root" / "W!n0&oO7."

**Comment by sasatefa2009** [comment]
@lucyoa nope, didn't work

**Comment by AlexHilgertBRA** [comment]
Fixed with #346 .  :)

**Comment by davidlj95** [comment]
I'm using [routersploit 3.1.0](https://github.com/threat9/routersploit/releases/tag/v3.1.0) that appears to have the [pull request that fixes this](#346) included, but my router isn't vulnerable and the exploit prints "Succesful authorization", despite isn't able to run any of the commands as hasn't been authenticated:

**Request**
```
POST /login.gch HTTP/1.1
Host: 192.168.0.1
User-Agent: python-requests/2.19.1
Accept-Encoding: gzip, deflate
Accept: */*
Connection: keep-alive
Cookie: _TESTCOOKIESUPPORT=1
Content-Length: 53
Content-Type: application/x-www-form-urlencoded

Frm_Logintoken=4&Username=root&Password=W%21n0%26oO7.
```

**Response**
```
HTTP/1.1 404 Not Found
Server: Mini web server 1.0 ZTE corp 2005.
Content-Type: text/html; charset=iso-8859-1
Accept-Ranges: bytes
Connection: close
Cache-Control: no-cache,no-store
Content-Length: 669

<HTML>
<HEAD><TITLE>404 Not Found</TITLE></HEAD>
<BODY BGCOLOR="#FFFFFF" TEXT="#000000" LINK="#2020ff" VLINK="#4040cc">
<H2>404 Not Found</H2>
The requested URL was not found on this server.
<!--
Padding so that MSIE deigns to show this error instead of its own canned one.
Padding so that MSIE deigns to show this error instead of its own canned one.
Padding so that MSIE deigns to show this error instead of its own canned one.
Padding so that MSIE deigns to show this error instead of its own canned one.
Padding so that MSIE deigns to show this error instead of its own canned one.
Padding so that MSIE deigns to show this error instead of its own canned one.
-->
```

That happens because [the code in the merged pull request that solved that](#346) was deleted [in a later](https://github.com/threat9/routersploit/blob/9380c04707da979f3081e6b3678c00f7f612bd95/routersploit/modules/exploits/routers/zte/zxv10_rce.py#L140) [commit](https://github.com/threat9/routersploit/commit/9380c04707da979f3081e6b3678c00f7f612bd95#diff-2657811e31e8e3c7b84be0f6303f19fb).

I have already added the code again and tested it, will link this issue in the pull request
